test/integration/patients/worklist/bulk-edit.js

```diff
@@ -674,7 +662,11 @@ context('Worklist bulk editing', function() {
         fx.data[3].attributes.created_at = testTsSubtract(2);
         fx.data[3].relationships.state = { data: { id: '33333' } };
 
-        fx.included.push(flowInclude);
+        fx.included.push({
+          id: '1',
+          type: 'flows',
+          attributes: _.extend({}, _.sample(fxFlows), { id: '1', name: 'Test Flow' }),
```

Comment:
we need to pull all of the `id`s from the `attributes`

Reply:
Ah ya, those have been removed now.